pkg/server/wire.go

```diff
@@ -345,6 +346,7 @@ var wireBasicSet = wire.NewSet(
 	grpcserver.ProvideService,
 	grpcserver.ProvideHealthService,
 	grpcserver.ProvideReflectionService,
+	objectdummyserver.ProvideDummyObjectServer,
```

Comment:
should this be under `wireTestSet`?

Reply:
I added it to the main wire.go so we can play with it e2e and e.g. create some simple plugin that uses the API. Thats also the reason for writing the collection to disk rather than keeping it memory - its going to retain the data between grafana restarts

Reply:
i am not married to this idea though, let me know if that does not sound good

Reply:
makes sense to me :+1: